lib/src/chrome.dart

```diff
@@ -14,21 +14,19 @@ const _linuxExecutable = 'google-chrome';
 const _macOSExecutable =
     '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome';
 const _windowsExecutable = r'Google\Chrome\Application\chrome.exe';
-var _windowsPrefixes = [
-  Platform.environment['LOCALAPPDATA'],
-  Platform.environment['PROGRAMFILES'],
-  Platform.environment['PROGRAMFILES(X86)']
-];
+const _windowsPrefixes = ['LOCALAPPDATA', 'PROGRAMFILES', 'PROGRAMFILES(X86)'];
 
 String get _executable {
+  final windowsPrefixes =
+      _windowsPrefixes.map((name) => Platform.environment[name]).toList();
   if (Platform.environment.containsKey(_chromeEnvironment)) {
     return Platform.environment[_chromeEnvironment];
   }
   if (Platform.isLinux) return _linuxExecutable;
   if (Platform.isMacOS) return _macOSExecutable;
   if (Platform.isWindows) {
     return p.join(
-        _windowsPrefixes.firstWhere((prefix) {
+        windowsPrefixes.firstWhere((prefix) {
```

Comment:
just to be picky as all hell – you that map expression below HERE. Don't create the var. Don't run it unless you're on windows!